eng/code-quality-reports/src/main/java/com/azure/tools/revapi/transforms/TransitiveCoreChangesTransform.java

```diff
@@ -51,7 +51,7 @@ public TransformationResult tryTransform(@Nullable E oldElement, @Nullable E new
             return TransformationResult.keep();
         }
 
-        if (!CORE_ARCHIVE.matcher(newArchive).matches()) {
+        if (!newArchive.startsWith("com.azure:azure-core:")) {
```

Comment:
nit: should strings like this be const values defined elsewhere.

Reply:
Since they're only used in one location they don't explicitly need to be as the compiler will change them to constants in the class file.